CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# How to contribute
+
+Thank you for your interest in contributing to the FeedHenry project. We want
+keep this process as easy as possible so we've outlined a few guidelines below.
+For more information about the FeedHenry project, visit
+[our website](http://feedhenry.org/).
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the FeedHenry team is available for technical advice or feedback.
+You can reach us at #feedhenry and #feedhenry-raincatcher on [Freenode IRC](https://freenode.net/) or the
+[feedhenry-dev list](https://www.redhat.com/mailman/listinfo/feedhenry-dev)
```

Comment:
Going to replace this with only `feedhenry-raincatcher`. It's also the mail that we're putting on the  `author` fields in package.jsons